app/services/bill-runs/cancel/delete-bill-run.service.js

```diff
@@ -0,0 +1,229 @@
+'use strict'
+
+/**
+ * Deletes a bill run, all its associated records and the its match in the Charging Module API
+ * @module DeleteBillRunService
+ */
+
+const BillModel = require('../../../models/bill.model.js')
+const BillLicenceModel = require('../../../models/bill-licence.model.js')
+const BillRunModel = require('../../../models/bill-run.model.js')
+const BillRunChargeVersionYearModel = require('../../../models/bill-run-charge-version-year.model.js')
+const BillRunVolumeModel = require('../../../models/bill-run-volume.model.js')
+const { db } = require('../../../../db/db.js')
+const ChargingModuleDeleteBillRunRequest = require('../../../requests/charging-module/delete-bill-run.request.js')
+const { calculateAndLogTimeTaken } = require('../../../lib/general.lib.js')
+const ReviewChargeVersionModel = require('../../../models/review-charge-version.model.js')
+const ReviewLicenceModel = require('../../../models/review-licence.model.js')
+const ReviewReturnModel = require('../../../models/review-return.model.js')
+
+/**
+ * Deletes a bill run, all its associated records and the its match in the Charging Module API
```

Comment:
you have an extra 'the' in here by the looks of it?

Reply:
💯 

But I'm about to do exactly the same refactor for sending a bill run. So, I promise 😇 I'll fix it in that PR rather than wasting you lovely approval!